subprojects/docs/src/docs/userguide/authoring-builds/more_about_tasks.adoc

```diff
@@ -524,13 +524,13 @@ The source templates that the task will be processing. Single files and collecti
 
 * `templateData`
 +
-For this example, we’re using a custom class to represent the model data. However, it does not implement `Serializable`, so we can’t use the `@Input` annotation. That’s not a problem as the properties within `TemplateData` - a string and a hash map with serializable type parameters - are serializable and can be annotated with `@Input`. We use `@Nested` on `templateData` to let Gradle know that this is a value with nested input properties.
+For this example, we’re using a custom class to represent the model data. However, it does not implement `Serializable`, so we can’t use the `@Input` annotation. That’s not a problem as the properties within `TemplateData` — a string and a hash map with serializable type parameters — are serializable and can be annotated with `@Input`. We use `@Nested` on `templateData` to let Gradle know that this is a value with nested input properties.
```

Comment:
❓ Why those dash are changed? Do they give a better output? Or is this something the editor did automatically?

Reply:
I changed hyphens into em dashes manually because hyphens are incorrect grammar but em dashes are correct grammar (and are used consistently throughout the documentation).  Please merge that correction along with the other typo fixes.

Reply:
Thanks for the great feedback. I will rebase the PR and merge the change as well. Sorry for not seeing the difference between both characters. 😅